### ISSUE TITLE
Pelease support the kaminari 1.0

### ISSUE THREAD

**Issue by fanjieqi** [Sology/smart_listing#126]
Excuse me, the smart_listing.gemspec has specified the kaminari ~> 0.17, Could you please support the kaminari 1.0?

**Comment by ndbroadbent** [comment]
Yes an update is really important for me, since I can't use the `maily_herald-webui` library at the moment. I also use `activeadmin`, which depends on `>= 1.0.1`.
